### PR TITLE
Jormungandr: Handle 'add_poi_infos[]' parameter + test

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVITIAII-2499

### CODE REVIEW

**Review by kadhikari — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/Places.py [hunk 1]
```diff
@@ -159,17 +159,17 @@ class PlaceUri(ResourceUri):
     def __init__(self, *args, **kwargs):
         ResourceUri.__init__(self, authentication=False, output_type_serializer=PlacesSerializer, *args, **kwargs)
         self.parsers["get"].add_argument("bss_stands", type=BooleanType(), default=True,
-                                         help="Show bss stands availability")
+                                         help="Deprecated - Use add_poi_infos[]=bss_stands")
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Places.py]: should this parm set to False?

File: source/jormungandr/jormungandr/interfaces/v1/Places.py [hunk 2]
```diff
@@ -233,12 +233,12 @@ def __init__(self, *args, **kwargs):
         self.parsers["get"].add_argument("start_page", type=int, default=0,
                                          help="The page number of the ptref result")
         self.parsers["get"].add_argument("bss_stands", type=BooleanType(), default=True,
-                                         help="Show bss stands availability")
+                                         help="Deprecated - Use add_poi_infos[]=bss_stands")
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Places.py]: and this one?

File: source/jormungandr/jormungandr/interfaces/common.py [hunk 3]
```diff
@@ -40,4 +40,14 @@
 }
 
 # When an emtpy string is passed, it deactivates all
-add_poi_infos_types = ('bss_stands', 'car_park', '')
+add_poi_infos_types = ('bss_stands', 'car_park', '', 'none')
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/common.py]: the comment is no longer valid :)

File: source/jormungandr/jormungandr/interfaces/common.py [hunk 4]
```diff
@@ -40,4 +40,14 @@
...
+
+
+def handle_poi_infos(add_poi_info_param, bss_stands_param):
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/common.py]: :+1:

File: source/jormungandr/jormungandr/interfaces/common.py [hunk 5]
```diff
@@ -40,4 +40,14 @@
...
+    if bss_stands_param and "bss_stands" not in add_poi_info_param:
+        add_poi_info_param.append("bss_stands")
+
+    if any(value in add_poi_info_param for value in ['bss_stands', 'car_park']):
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/common.py]: return any(value in add_poi_info_param for value in ['bss_stands', 'car_park'])

**Review by xlqian — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**